src/store/getters.ts

```diff
@@ -36,10 +36,15 @@ export default {
   },
 
   /** Is True if there are any pending tasks or filings. */
-  hasBlocker (state: StateIF, getters): boolean {
-    let blocker = (state.hasBlockerTask || state.hasBlockerFiling || state.isCoaPending)
+  hasBlockerExceptStaffApproval (state: StateIF): boolean {
```

Comment:
created a new getter like before , and used this to avoid conversion menu disable, Let me know if we have other ways to do it

Reply:
This is a special case that might get more and complicated over time.

What if you copied `(state.hasBlockerTask || state.hasBlockerFiling || state.isCoaPending)` into the allowable actions case (instead of using the global getter)? (You will also have to update the global getter.) I think this might be cleaner.

Reply:
Have to use those in two places, To avoid code duplicate , I used as a getter